src/main/java/org/folio/service/orders/PurchaseOrderLineService.java

```diff
@@ -499,18 +499,19 @@ public Future<Void> validateAndNormalizeISBNCommon(List<CompositePoLine> composi
       });
   }
 
-  public Future<Void> updateSearchLocations(CompositePoLine poLine, RequestContext requestContext) {
-    return updateSearchLocations(HelperUtils.convertToPoLine(poLine), requestContext);
```

Comment:
root cause of not populating searchLoctionIds was creating new object by HelperUtils.convertToPoLine(poLine), so searchLocationIds are set to this new object instead of original poLine